toolkit/javascripts/report-a-problem.js

```diff
@@ -1,5 +1,5 @@
 // REDUCED VARIANT OF https://github.com/alphagov/static/blob/master/app/assets/javascripts/report-a-problem.js
-(function() {
+;(function() {
```

Comment:
Interesting! I'm not familiar with this JS syntax myself but I notice not all the `.js` files in this folder have that semi colon - when do/don't we need it?

Reply:
We should probably start+end all of our JS files with a semi colon to allow them to be safely concatenated between other JS files. At least that's the conclusion I've drawn from this fix and a quick read around.

Reply:
We should really have gulp do this or actually check all files for issues

Reply:
This was new on me too. Definitely worth a bit of a note out. We should probably have a coding standard for JS, and maybe if we polish it enough it could be in the GDS Way.